webpack.config.renderer.dev.js

```diff
@@ -192,7 +192,10 @@ module.exports = merge.smart(baseConfig, {
 
     new webpack.LoaderOptionsPlugin({
       debug: true
-    })
+    }),
+    new webpack.DefinePlugin({
+      __static: `"${path.join(configurator.projectDir, "static").replace(/\\/g, "\\\\")}"`,
```

Comment:
I can't seem to run this in dev-mode after a clean `git clone` & `yarn install`:
```
$ npm run start-server-dev

> bloomrpc@1.2.1 start-server-dev /home/bjaglin/git/projects/bloomrpc
> cross-env NODE_ENV=development node --trace-warnings ./node_modules/webpack-dev-server/bin/webpack-dev-server --config webpack.config.renderer.dev.js

/home/bjaglin/git/projects/bloomrpc/webpack.config.renderer.dev.js:197
      __static: `"${path.join(configurator.projectDir, "static").replace(/\\/g, "\\\\")}"`,
                              ^

ReferenceError: configurator is not defined
    at Object.<anonymous> (/home/bjaglin/git/projects/bloomrpc/webpack.config.renderer.dev.js:197:31)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at WEBPACK_OPTIONS (/home/bjaglin/git/projects/bloomrpc/node_modules/webpack-cli/bin/convert-argv.js:115:13)
    at requireConfig (/home/bjaglin/git/projects/bloomrpc/node_modules/webpack-cli/bin/convert-argv.js:117:6)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

Reply:
Good catch! i forgot to replace that variable

Reply:
@bjaglin it's now fixed